lib/morandi/profiled_pixbuf.rb

```diff
@@ -30,6 +30,15 @@ def initialize(*args)
     end
 
     super(*args)
+  rescue Gdk::PixbufError::CorruptImage => e
+    if args[0].is_a? String
+      temp_path =  Tempfile.new.path
+      pixbuf.save(temp_path, 'jpeg')
```

Comment:
ups :)

Reply:
what's the oops? That you've got a variable reference to `pixbuf`?

Reply:
Now I have, yes :)

Reply:
Will the TempFile get cleaned up, do you think?

Reply:
It will be cleaned but i think we should explicity close it. As good practice says. Thx